scripts/fa5upgrade.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+echo "Please enter your FontAwesome5 npm token:"
+
+read fa5_token
+
+echo "Setting up npm config"
+
+npm config set "@fortawesome:registry" https://npm.fontawesome.com/
+npm config set "//npm.fontawesome.com/:_authToken" $fa5_token
+
+echo "Installing FontAwesome5 Pro"
+
+npm install @fortawesome/fontawesome-pro --no-shrinkwrap
+
+echo "Copying font files"
+
+cp ./node_modules/@fortawesome/fontawesome-pro/webfonts/fa-brands-400.ttf Fonts/FontAwesome5_Brands.ttf
+cp ./node_modules/@fortawesome/fontawesome-pro/webfonts/fa-light-300.ttf Fonts/FontAwesome5_Light.ttf
```

Comment:
It's not ideal to install these to the `node_modules` folder IMHO as it is usually not checked in to git and thus wouldn't work out of the box if you're more than one dev. Would it be possible to either use the fonts from the `@fortawesome/fontawesome-pro` package directly or output them to the main project's directory?

Reply:
Not sure, the API requires direct access to the files but maybe the FontAwesome5Pro module could point backwards? I'll look into it!